lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -62,12 +62,20 @@ class LineChartPainter extends AxisChartPainter<LineChartData> {
     PaintHolder<LineChartData> holder,
   ) {
     final data = holder.data;
+    if (holder.boundingBox != null) {
+      canvasWrapper
+        ..saveLayer(
+          Offset.zero & canvasWrapper.size,
+          Paint(),
```

Comment:
It's better to save this Paint() instance to prevent it from being re-created in every frame.

Reply:
good catch 👍 Added a `_clipPaint` property and used it here

Reply:
Nice!
Thanks